src/views/validated-designs/guide/index.tsx

```diff
@@ -27,6 +28,11 @@ import type { MDXRemoteSerializeResult } from 'lib/next-mdx-remote'
 import type { OutlineLinkItem } from 'components/outline-nav/types'
 import { ProductSlug } from 'types/products'
 
+// Keep in sync HVD PDF location
+const PDF_BASE_URL = 'https://d2cn2jw7bw0rn4.cloudfront.net/'
```

Comment:
Not happy with this, but not sure how to best share this across both repos and didn't want to get too into the weeds for this small of a feature.